src/components/Modal.css

```diff
@@ -0,0 +1,44 @@
+.modal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+}
+
+.modal-content {
+  background-color: rgba(0, 0, 0, 0.8);
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 40%;
+  box-shadow: 0 5px 8px 0 rgba(0, 0, 0, 0.2);
+  padding: 20px;
+  height: 400px;
+  overflow: scroll;
+  color: #fff;
+  font-family: arial;
+  font-size: 14px;
+  position: fixed;
+}
+
+.show {
+  display: block;
+}
+
+.hide {
+  display: none;
+}
+
+.close-btn {
+  color: #fff;
+  font-size: 30px;
+}
+
+.close-btn:hover {
+  cursor: pointer;
+  color: rgb(197, 54, 54);
```

Comment:
I'd recommend your hover color to be white and your non-hover color to be white with an opacity of 50-75%.

Reply:
Done